go/air/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.12.8 AS buildenv
+
+ENV GO111MODULE=on
```

Comment:
@aofei this seems to be required. why is that on `go` **1.12** ?

I have the same issue with https://github.com/gogf/gf

Reply:
Currently, the default value for `GO111MODULE` is "auto" (same for the upcoming Go 1.13). In this option, modules will only be enabled if the working directory isn't in `GOPATH` and contains a `go.mod` file (note that the `GOPATH` condition will be ignored in Go 1.13). Therefore, we usually explicitly set `GO111MODULE` to "on" to avoid situations where modules aren't enabled.

One day (I guess it's Go 1.14), `GO111MODULE` will be defaulted to "on".